inst/tutorials/07-probability1/07-probability1.Rmd

```diff
@@ -72,18 +72,25 @@ question("Out of all possible anagrams for the word CHAIR (including the origina
 
 Let A and B be two independent variables that take on only the values 0 and 1. If P(A=1, B=1)=0.12, P(A=0, B=1)=0.28, and P(A=1)=0.3, what is P(A=0, B=0)?
 
-```{r indep, exercise = TRUE}
+
+```{r indep_chunk, exercise = TRUE}
 
 ```
 
-```{r indep-check}
-grade_result(pass_if(~ identical(.result, 0.42)))
+
+```{r indep}
+question_text(
+  "Answer:",
+  answer("0.42", correct = TRUE),
+  answer(".42", correct = TRUE),
+  allow_retry = TRUE
+)
```

Comment:
@MafaldaPratas I changed here from the open coding chuck style to the answer box to be consistent with the previous questions.